pages/image/image.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href='../../index.css'>
+  <link rel="stylesheet" href='./image.css'>
+  <link rel="stylesheet" href="../avatar/avatar.css">
+  <title>starUI🌟 | Image</title>
+</head>
+<body>
+  <div class="body-center">
+    <nav class='navbar'>
+      <div class="logo">
+        <h2>starUI🌟</h2>
+      </div>
+      <div class="right-nav">
+        <ul>
+          <li>Docs</li>
+          <li>Github</li>
+          <li>Twitter</li>
+        </ul>
+      </div>
+    </nav>
+    <main class="container-flex">
+      <aside class="aside">
+        <h4 class='underline'>Get Started</h4>
+        <ul>
+          <li><a href="../introduction/introduction.html">Introduction</a></li>
+          <li><a href="../colors/colors.html">Colors</a></li>
+          <li><a href="../typography/typography.html">Typography</a></li>
+          <li><a href="../conclusion/conclusion.html">Conclusion</a></li>
+        </ul>
+        <h4 class="underline">Components</h4>
+        <ul>
+          <li><a href="../avatar/avatar.html">Avatar</a></li>
+          <li><a href="../alert/alert.html">Alert</a></li>
+          <li><a href="../badge/badge.html">Badge</a></li>
+          <li><a href="../button/button.html">Button</a></li>
+          <li><a href="./cards.html">Card</a></li>
+          <li><a class="active" href="./image.html">Image</a></li>
+          <li><a>Input</a></li>
+        </ul>
+      </aside>
+      <section class="section">
+        <article class="section-content underline">
+          <h2>Image</h2>
+          <p class="small-text">
+            Images make the website more interactive, as they display more context to the user.
+          </p>
+        </article>
+        <article class="section-content underline">
+          <h2>Square Image</h2>
+          <p class="small-text">Square images as the name suggest are squared with a default dimension of 150*150. </p>
+          <div class="example-code">
+            <div class="example-container-flex">
+              <img src="../../assets/image1.jpg" alt="" class="square-img">
```

Comment:
**Fix:** Give value to the _"alt"_ attribute. It helps with accessibility.

Reply:
Fixed it ,Vishal .